libraries/RC_Channel/RC_Channel.cpp

```diff
@@ -1261,7 +1261,7 @@ RC_Channel *RC_Channels::find_channel_for_option(const RC_Channel::aux_func_t op
 // duplicate_options_exist - returns true if any options are duplicated
 bool RC_Channels::duplicate_options_exist()
 {
-    uint8_t auxsw_option_counts[256] = {};
+    uint8_t auxsw_option_counts[uint32_t(RC_Channel::AUX_FUNC::AUX_FUNC_END)] = {};
```

Comment:
best to change this to an alloc/free, as this is a lot of stack space

Reply:
I change to check with a bit